### DISCUSSION TITLE
Is there a setting to remove the top 30 rows?

### DISCUSSION THREAD

**Discussion by ghost** [didierterrien/Power-BI-Sidetools#13] [Ideas]
I would like to see all the rows...Because some times the error happens even further....

**Comment by didierterrien** [comment]
Hello,
You may add a line in your measure formula : 
//VAR DBG_ROWS_LIMIT = YourLimit
Replace YourLimit with the number of rows you want to see. The highest it will be, the longer it will take to load the results.
I personally prefer to build a context with the appropriate filter so I avoid scrolling and scrolling and missing some errors : 
//VAR DBG_MAIN_CONTEXT = FILTER( 'YourTable', 'YourTable'[YourColumn] = "YourValue" )
Replace the formula behind DBG_MAIN_CONTEXT with any formula returning a table.

I plan to export the result of any debug table without limit to an Excel or CSV file.
